fir_notifications/methods/email.py

```diff
@@ -0,0 +1,84 @@
+import markdown2
+from django import forms
+from django.conf import settings
+from django.core import mail
+from django.utils.translation import ugettext_lazy as _
+
+
+from fir_notifications.methods import NotificationMethod
+from fir_notifications.methods.utils import request
+from fir_plugins.links import registry as link_registry
+
+
+class EmailMethod(NotificationMethod):
+    use_subject = True
+    use_description = True
+    name = 'email'
+    verbose_name = 'Email'
+
+    def __init__(self):
+        super(EmailMethod, self).__init__()
+        if hasattr(settings, 'EMAIL_FROM') and settings.EMAIL_FROM is not None:
+            self.server_configured = True
+        if 'djembe' in settings.INSTALLED_APPS:
+            self.options['certificate'] = forms.CharField(required=False, label=_('Certificate'),
+                                                          widget=forms.Textarea(attrs={'cols': 60, 'rows': 15}),
+                                                          help_text=_('Encryption certificate in PEM format.'))
+
+    def send(self, event, users, instance, paths):
```

Comment:
Where can I put forms and templates used to configure S/MIME? `fir_email` is a Python module, not a Django app, so if I put these files in it, they won't be i18n ready. In `fir_plugins` ? in `incidents`?